pyccl/halos/halo_model.py

```diff
@@ -247,6 +247,53 @@ def I_1_1(self, cosmo, k, a, prof):
         uk = prof.fourier(cosmo, k, self._mass, a).T
         return self._integrate_over_mbf(uk)
 
+    def I_1_3(self, cosmo, k, a, prof, *, prof2=None, prof_2pt, prof3=None):
+        """ Solves the integral:
+
+        .. math::
+            I^1_3(k,a|u_2, v_1, _v2) = \\int dM\\,n(M,a)\\,b(M,a)\\,
+            \\langle u_2(k,a|M) v_1(k',a|M) v_2(k',a|M)\\rangle,
+
+        approximated to
+        .. math::
+            I^1_3(k,a|u_2, v_1, _v2) = I^1_1(k,a|u_2) I^1_2(k',a|v_1, v_2)
```

Comment:
Humm, I don't think this equation is correct. What you're approximating is <uvv> ~ <u><vv>, right? where the angle brackets are an average over profile realisations, not over mass

Reply:
I think now is correct. I wasn't sure what you meant about the average.

Reply:
Don't think so. Let me suggest something

Reply:
Suggestion accepted. Now I see what you meant. Thanks!